src/Routes/routes.php

```diff
@@ -11,5 +11,7 @@
 
     // Setup route for searching recipes by category
     $app->get('/recipes/byCategory/{categoryId}', '\App\Services\RecipeHandler:getRecipesByCategoryId');
+
+    $app->get('/recipes/{recipeId}', '\App\Services\RecipeHandlerId:getRecipeById');
```

Comment:
use $app-> group for the recipes
ex:
 $app->group('/students', function () use ($app): void {
        $app->get('/', User\GetAllStudents::class);
        $app->get('/data', User\GetStudentByEmail::class);
        $app->get('/disciplines', User\getDisciplinesByUserEmail::class);
        $app->get('/grades', User\GetAllGradesByUserEmail::class);
    });